db/migrate/20140407202158_site_setting_comma_to_pipe.rb

```diff
@@ -5,7 +5,7 @@ def up
     execute <<SQL
       UPDATE site_settings
       SET value = replace(value, ',', '|')
-      WHERE name = 'white_listed_spam_host_domains'
+      WHERE name = 'allowed_spam_host_domains'
```

Comment:
I do not think we should be changing migrations too. On the other hand, I doubt there is anyone with a schema that old looking to migrate.